www/listOfMetadata.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+use SimpleSAML\Metadata\MetaDataStorageHandler;
+use \SimpleSAML\Module\perun\Disco;
+
+$metadataHandler = MetaDataStorageHandler::getMetadataHandler();
+
+$metadata = $metadataHandler->getList();
+$filteredMetadata = Disco::doFilter($metadata);
+
+$data = [];
+
+foreach ($filteredMetadata as $metadata) {
+    $item = [];
+    $item['entityid'] = $metadata['entityid'];
+    $item['name'] = $metadata['name'];
```

Comment:
What data should we provide? I would suggest adding the "description" field as well.